src/lib/hooks/swap/useSwapDefaults.ts

```diff
@@ -0,0 +1,63 @@
+import { NativeCurrency } from '@uniswap/sdk-core'
+import { nativeOnChain } from 'constants/tokens'
+import { useUpdateAtom } from 'jotai/utils'
+import { DefaultAddress } from 'lib/components/Swap'
+import useActiveWeb3React from 'lib/hooks/useActiveWeb3React'
+import { useToken } from 'lib/hooks/useCurrency'
+import { Field, Swap, swapAtom } from 'lib/state/swap'
+import { useCallback, useEffect, useState } from 'react'
+
+function useDefaultToken(
+  defaultAddress: DefaultAddress | undefined,
+  chainId: number | undefined
+): ReturnType<typeof useToken> | NativeCurrency {
```

Comment:
I think this return type is already aliased to `Currency`, imported from `@uniswap/sdk-core`. Use that instead.